sdk/iot/Azure.Iot.Hub.Service/samples/IotHubClientSamples/ModuleIdentityLifecycleSamples.cs

```diff
@@ -24,40 +26,37 @@ public async Task RunSampleAsync()
             string moduleId = $"module{Random.Next(MaxRandomValue)}";
 
             // Create a DeviceIdentity.
-            DeviceIdentity deviceIdentity = await CreateDeviceIdentityAsync(deviceId);
+            await CreateDeviceIdentityAsync(deviceId);
 
             // Create a ModuleIdentity.
```

Comment:
Are these comments really adding anything? Seems like they could be removed

Reply:
I think they can help follow the logic of the test ... why remove them?

Reply:
Don't they seem a bit redundant? "// Create a DeviceIdentity" doesn't add anything that "CreateDeviceIdentityAsync(...)" hasn't already said

Reply:
I am sorry, I am going to have to disagree, I think comments in code are useful for future authors of the code. They may seem redundant, but I don't see any harm in having step by step comments in the code even though the action can be identified by the code. if we follow that logic, all comments in code are redundant. am I making sense? One can argue that code comments are suppose to explain the reasoning behind the logic that was employed in the code, but they are also suppose to help follow what is happening in the block of code.